### DISCUSSION TITLE
blog/posts/230111-gatsby-1/

### DISCUSSION THREAD

**Discussion by giscus** [ricale/blog#121] [Giscus]
# blog/posts/230111-gatsby-1/

쓰던 블로그 서비스가 안드로이드 모바일에서 광고를 삽입하는 것을 발견했다. (광고는 다시 없어졌다. 아마 운영자의 테스트였던 것 같다.) 개인적으로 개인 개발 블로그에 광고가 들어가는 걸 원치 않으므로 서비스를 바꿔야 겠다는 생각이 들었다. 그리고…

https://ricale.kr/blog/posts/230111-gatsby-1/

**Comment by yanggwi** [comment]
참고 완료!
